link_release.sh

```diff
@@ -34,6 +34,10 @@ NEW_RELEASE=$(jq \
   --arg dir_mac "${DIR_MAC}" \
   --arg dir_win "${DIR_WIN}" \
   \
+  --arg sha_linux "${sha_linux}" \
+  --arg sha_macos "${sha_macos}" \
+  --arg sha_windows "${sha_windows}" \
```

Comment:
pipe in as argument just for clarity, don't think these --arg are essential but are nice